.storybook/addons/addon-theme/register.js

```diff
@@ -58,7 +58,12 @@ register(namespace, api =>
   addPanel(`${namespace}/panel`, {
     title,
     render: ({ active }) => (
-      <Theme active={active} api={api} channel={getChannel()} />
+      <Theme
+        key="storybook/theme/panel"
```

Comment:
Nice! I wasn't familiar with this convention, where can I find more information about it? Perhaps in a follow-up, we update the `namespace`

Reply:
You can see the keys of other storybook panels if you inspect the storybook environment using React DevTools:

<img width="306" alt="Screen Shot 2020-05-21 at 8 20 11 AM" src="https://user-images.githubusercontent.com/9057921/82563066-fc826080-9b3b-11ea-99ee-68960a7f86dd.png">

I don't recommend trying to edit those keys though.